### PR TITLE
Updating CHANGELOG.md to version 10.1

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 10.1
+* Add sponsor logo dimensions to answers (by bumping the content-api-models dependency)
```
Comment (LATaylor-guardian) [inline CHANGELOG.md]: Not sure you need the `(by bumping the content-api-models dependency)`, this is transparent to the client.

Comment (shtukas) [inline CHANGELOG.md]: Done!

Comment (LATaylor-guardian) [inline CHANGELOG.md]: Also, `to answers`, do you mean Sponsorship model?

**Comment by LATaylor-guardian** [comment]
Other than my comment 👍

File: CHANGELOG.md (hunk 1 above)
Comment (shtukas) [inline CHANGELOG.md]: Yes, sorry. I didn't know how to best formulate it.

**Pull request merged**